libs/community/langchain_community/chat_models/baidu_qianfan_endpoint.py

```diff
@@ -613,6 +617,15 @@ def _stream(
                         role="assistant",
                         additional_kwargs=additional_kwargs,
                         usage_metadata=msg.usage_metadata,
+                        tool_call_chunks=[
```

Comment:
Should we add this to `_astream` below, as well?

Reply:
oh, I forgot that